app/views/categories/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="is-flex flex-direction-column flex-grow-1 has-background-color">
+  <header class="header">
+    <button class="button has-text-white">
+      <span class="material-icons-outlined">home</span>
+    </button>
+    <div class="is-flex flex-grow-1 justify-content-center">
+      <h1 class="title">Categories</h1>
+    </div>
+    <%= button_to destroy_user_session_url, method: :delete, class: 'button has-text-white' do %>
+      <span class="material-icons-outlined">logout</span>
+    <% end %>
+  </header>
+ <%= render @categories %>
+</div>
+<footer class="p-4 is-secondary">
+  <%= link_to 'Add a new category', new_category_path, class: "button is-outlined" %>
+</footer>
```

Comment:
- [ ] **[OPTIONAL]** When there are no categories created, I suggest you add a friendly message that tells user that there are no categories added yet. This will make your app more user-friendly.